docs/auth/Single Sign-On via Device Browser.adoc

```diff
@@ -0,0 +1,105 @@
+== Single Sign-On (SSO) via the Device Browser
+
+The Auth SDK uses OpenID https://github.com/openid/AppAuth-Android[AppAuth library] and implements OpenID's http://openid.net/specs/openid-connect-core-1_0.html#CodeFlowAuth[auhtorization code flow] to achieve SSO via the device browser.
```

Comment:
Maybe this can be re-phrased to the following. I think we can leave out the reference to AppAuth,

The Aerogear Auth SDK uses the OpenID Connect http://openid.net/specs/openid-connect-core-1_0.html#CodeFlowAuth[Authorization Code Flow] to achieve SSO via the device browser.

Reply:
Ya sure sounds good to me.  Oh and sorry forgot to respond to @ciaranRoche and your suggestion above too.  I'm thinking maybe it would be better to include the best practices stuff in the main auth doc? What are your thoughts on this?

Reply:
Yeah main docs is best for this since it's not specific to SSO, so +1 from me to have it there instead.